src/SpiAnalyzer.cpp

```diff
@@ -58,7 +58,14 @@ void SpiAnalyzer::AdvanceToActiveEnableEdgeWithCorrectClockPolarity()
     for( ;; )
     {
         if( IsInitialClockPolarityCorrect() == true ) // if false, this function moves to the next active enable edge.
+        {
+            if( mEnable )
+            {
+                FrameV2 frame_v2_start_of_transaction;
+                mResults->AddFrameV2( frame_v2_start_of_transaction, "Enable", mCurrentSample, mCurrentSample + 1 );
```

Comment:
```suggestion
                mResults->AddFrameV2( frame_v2_start_of_transaction, "enable", mCurrentSample, mCurrentSample + 1 );
```

My only suggestion would be to use lowercase for the type to be consistent with the others.